comp/core/gui/guiimpl/agent.go

```diff
@@ -153,19 +153,19 @@ func makeFlare(w http.ResponseWriter, r *http.Request, flare flare.Component) {
 	filePath, e := flare.Create(nil, nil)
 	if e != nil {
 		w.Write([]byte("Error creating flare zipfile: " + e.Error()))
-		log.Errorf("Error creating flare zipfile: " + e.Error())
+		log.Errorf("Error creating flare zipfile: %s", e.Error())
 		return
 	}
 
 	res, e := flare.Send(filePath, payload.CaseID, payload.Email, helpers.NewLocalFlareSource())
 	if e != nil {
-		w.Write([]byte("Flare zipfile successfully created: " + filePath + "<br><br>" + e.Error()))
-		log.Errorf("Flare zipfile successfully created: " + filePath + "\n" + e.Error())
+		w.Write([]byte("Flare zipfile unsuccessfully created: " + filePath + "<br><br>" + e.Error()))
+		log.Errorf("Flare zipfile unsuccessfully created: %s\n%s", filePath, e.Error())
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏The flare was successfully created (a few lines above), but failed to be sent, the original message could be clearer but this change is not correct

Reply:
Thank you for spotting. I agree, that the original message could improve. I would revert the message change. We can work on improving the message on separate PR.